osquery/tests/integration/tables/curl_certificate.cpp

```diff
@@ -19,31 +19,23 @@ namespace osquery {
 class curlCertificate : public IntegrationTableTest {};
 
 TEST_F(curlCertificate, test_sanity) {
-  // 1. Query data
-  // QueryData data = execute_query("select * from curl_certificate");
-  // 2. Check size before validation
-  // ASSERT_GE(data.size(), 0ul);
-  // ASSERT_EQ(data.size(), 1ul);
-  // ASSERT_EQ(data.size(), 0ul);
-  // 3. Build validation map
-  // See IntegrationTableTest.cpp for avaialbe flags
-  // Or use custom DataCheck object
-  // ValidatatioMap row_map = {
-  //      {"hostname", NormalType}
-  //      {"common_name", NormalType}
-  //      {"organization", NormalType}
-  //      {"organization_unit", NormalType}
-  //      {"serial_number", NormalType}
-  //      {"issuer_common_name", NormalType}
-  //      {"issuer_organization", NormalType}
-  //      {"issuer_organization_unit", NormalType}
-  //      {"valid_from", NormalType}
-  //      {"valid_to", NormalType}
-  //      {"sha256_fingerprint", NormalType}
-  //      {"sha1_fingerprint", NormalType}
-  //}
-  // 4. Perform validation
-  // validate_rows(data, row_map);
+  QueryData data = execute_query("select * from curl_certificate where hostname=\"www.facebook.com\"");
```

Comment:
We should use a better hostname, and merge. Not sure what, maybe github, since the code is all being cloned from there.

Reply:
@directionless I changed the hostname to github :) Seems more appropriate